openpgp.d.ts

```diff
@@ -337,13 +338,20 @@ interface Config {
   rejectMessageHashAlgorithms: Set<enums.hash>;
   rejectPublicKeyAlgorithms: Set<enums.publicKey>;
   rejectCurves: Set<enums.curve>;
+  hardwareKeys?: HardwareKeys;
```

Comment:
```suggestion
  hardwareKeys: HardwareKeys | null;
```
All fields are set in `Config` -- `PartialConfig` (just below) declares everything optional

Reply:
Hey @larabr!
That makes sense indeed - applied as proposed.